computer_assisted_intervention/video_seg.ipynb

```diff
@@ -369,7 +369,14 @@
    "source": [
     "## Normal training loop\n",
     "\n",
-    "A normal training loop will be excuted. Meanwhile, the iou value of each epoch will be recorded as the accuracy."
+    "A normal training loop will be excuted. Meanwhile, the iou value of each epoch will be recorded as the accuracy. \n",
+    "There is a pretrained bundle for surgical tool segmentation task. This model can be loaded to the network by adding codes to the block below:\n",
```

Comment:
I would suggest to change it to:
"There is a pretrained bundle for surgical tool segmentation task in the MONAI model-zoo, the model can be automatically downloaded and loaded to the network by adding codes to the block below:"
What do you think?

Thanks.

Reply:
Hi @Nic-Ma ,
    I think it's ok and have updated the description.
Thanks,
Bin